src/Microsoft.Identity.Abstractions/ManagedIdentity/ManagedIdentityDescription.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System.ComponentModel;
+
+namespace Microsoft.Identity.Abstractions
+{
+    /// <summary>
+    /// Data object to hold the description of a managed identity for an application to use for authentication.
+    /// </summary>
+    public class ManagedIdentityDescription
+    {
+        /// <summary>
+        /// Gets or sets the type of managed identity (and if applicable the unique id) as defined in
+        /// <see cref="ManagedIdentityType"/>.Defaults to <see cref="ManagedIdentityType.SystemAssigned"/> if not set.
+        /// </summary>
+        [DefaultValue(ManagedIdentityType.SystemAssigned)]
+        public ManagedIdentityType IdType { get; set; }
```

Comment:
Should we initialize this property to ManagedIdentityType.SystemAssigned ? since this is the default?

Reply:
Isn't that what the default value does? I wasn't sure if I was allowed to have a constructor since you said inheritance wasn't allowed, I wasn't sure how simple the data object class needed to be.

Reply:
The property still needs to be set the default value is for the editor/tools.

Reply:
but the test passes where I init with no values, is that not the behavior I could expect in real application?